app/models/spell.rb

```diff
@@ -77,13 +80,12 @@ def list_spell_requirements
       end
     end
 
-    if display.include?("f")
-      display = display.split("").sort.reverse!
-    else
-      display = display.split("")
-    end
+    display = if display.include?("f")
```

Comment:
I like this syntax, it makes much more sense as it says the variable is the direct result of the if statements.